### ISSUE TITLE
Reactive-Extensions or ReactiveX?

### ISSUE THREAD

**Issue by jasonwilliams200OK** [Reactive-Extensions/RxJS#1029]
Are these two the same organizations:

* https://github.com/Reactive-Extensions/
* https://github.com/ReactiveX/

Also looking readme at https://github.com/ReactiveX/RxJS, it seems like this repository is / will be deprecated. If that is true, will this repository be removed at some point?

**Comment by dralletje** [comment]
Yes, at some point this will be superseded by ReactiveX, but till that day (and you'll notice, trust me) you should use this repo. This repo is still maintained and a lot of people use it :)
